app/views/layouts/_header.html.slim

```diff
@@ -1,20 +1,33 @@
 header
-  nav.navbar.navbar-inverse.navbar-fixed-top[role="navigation"]
-    .container-fluid
-      .navbar-header
-        button.navbar-toggle.collapsed[type="button" data-toggle="collapse" data-target="#menu-collapse"]
-          span.sr-only
-            | Toggle navigation
-            span.icon-bar
-            span.icon-bar
-            span.icon-bar
-        a.navbar-brand[href="/"]
-          span
-            | New
-          | Vasjuky
-        #menu-collapse
-          ul.user-menu.pull-right
-            li
-              button.btn.btn-primary
-                | Login with Facebook
-                / Place your facebook login code up here
+  .row
```

Comment:
Are You sure about it?
I think we can use nav tag without row and container classes on top levels
See http://getbootstrap.com/components/#navbar and HTML source of http://getbootstrap.com/examples/starter-template/